common/src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -53,6 +53,8 @@
                 hints = @QueryHint(name = BehaviorGroup.HINT_PASS_DISTINCT_THROUGH, value = "false")),
         @NamedQuery(name = "BehaviorGroup.countByDisplayNameAndAccountId",
                 query = "SELECT COUNT(*) FROM BehaviorGroup b WHERE b.displayName = :displayName AND b.accountId = :accountId"),
+        @NamedQuery(name = "BehaviorGroup.countByDisplayAccountId",
+                query = "SELECT COUNT(*) FROM BehaviorGroup b WHERE b.displayName = :displayName"),
```

Comment:
We need to add a check for the accountId here
```suggestion
                query = "SELECT COUNT(*) FROM BehaviorGroup b WHERE b.displayName = :displayName" and b.accountId IS NULL),
```

Reply:
added. thanks!